src/main/java/at/hannibal2/skyhanni/mixins/transformers/MixinRendererLivingEntity.java

```diff
@@ -0,0 +1,25 @@
+package at.hannibal2.skyhanni.mixins.transformers;
+
+import at.hannibal2.skyhanni.utils.EntityOutlineRenderer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.entity.RendererLivingEntity;
+import net.minecraft.entity.EntityLivingBase;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(RendererLivingEntity.class)
+public class MixinRendererLivingEntity {
+    @Redirect(method = "setScoreTeamColor", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/renderer/GlStateManager;color(FFFF)V"))
+    public void setOutlineColor(float colorRed, float colorGreen, float colorBlue, float colorAlpha, EntityLivingBase entity) {
+        Integer i = EntityOutlineRenderer.Companion.getCustomOutlineColor(entity);

```

Comment:
Please use a hook instead of writing more than one line per mixin method. This helps to read stack traces enormously